src/rasterizer.cpp

```diff
@@ -48,7 +48,8 @@ namespace gs {
         float scaling_modifier,
         bool packed,
         bool antialiased,
-        RenderMode render_mode) {
+        RenderMode render_mode,
+        const gs::BoundingBox* bounding_box) {
```

Comment:
This is okay for now. 
I would like to separate the training from viewer rendering eventually. The training performance should be as high as possible and the training implementation should be uncluttered and minimal.

However, we don't have that yet implement so let's keep it in mind.